### PR TITLE
Create CODE_OF_CONDUCT.md

### PR DESCRIPTION
**Pull request by maxkadel**
This uses the default Contributor Covenant Code of Conduct.

Connected to #4

### CODE REVIEW

File: CODE_OF_CONDUCT.md [hunk 1]
```diff
@@ -0,0 +1,128 @@
...
+an individual is officially representing the community in public spaces.
+Examples of representing our community include using an official e-mail address,
+posting via an official social media account, or acting as an appointed
+representative at an online or offline event.
+
+## Enforcement
+
+Instances of abusive, harassing, or otherwise unacceptable behavior may be
+reported to the community leaders responsible for enforcement at
+mkadel@princeton.edu.
```
Comment (escowles) [inline CODE_OF_CONDUCT.md]: We should also say that Princeton staff can also report to their supervisor, the Title IX Coordinator (https://sexualmisconduct.princeton.edu/coordinator), or the Ombuds Office (https://ombuds.princeton.edu/contact-us) as appropriate.

Comment (maxkadel) [reply CODE_OF_CONDUCT.md]: Done.

**Comment by Beck-Davis** [comment]
The consequences from number 2 (warning) and number 3 (temporary ban) seem like they are communicating the same things, but number 3 is much clearer. If they are communicating the same thing, I'd suggest using the wording from the temporary ban consequences in the warning consequences.

This phrasing from the 2. Warning - Consequences is not as clear to me: 
`No
interaction with the people involved, including unsolicited interaction with
those enforcing the Code of Conduct, for a specified period of time.`

But I think it is communicating the same as this phrasing from 3. Temporary Ban - Consequences:
`No public or
private interaction with the people involved, including unsolicited interaction
with those enforcing the Code of Conduct, is allowed during this period.`

**Comment by sandbergja** [comment]
@kevinreiss @tpendragon @kelynch @escowles Are there any updates on this work from your recent meetings?  Thanks!

**Comment by tpendragon** [comment]
> @kevinreiss @tpendragon @kelynch @escowles Are there any updates on this work from your recent meetings? Thanks!

Yeah we're gonna use lsupport, I just haven't had a chance to update the email. Feel free if you have a second.

**Comment by tpendragon** [comment]
> @kevinreiss @tpendragon @kelynch @escowles Are there any updates on this work from your recent meetings? Thanks!

We just decided to use lsupport, which y'all have updated.

**Comment by escowles** [comment]
@Beck-Davis I think the difference between # 2 and # 3 is that # 2 says "No
interaction with the people involved" and # 3 says "A temporary ban from any sort of interaction or public communication with the community". So # 2 is just not interacting with the people directly involved in the incident, but # 3 is not interacting with the community at all.

**Comment by kevinreiss** [comment]
@escowles per your last comment do we need more discussion or can this be approved?

**Comment by escowles** [comment]
@kevinreiss I see the difference between # 2 and # 3 now, but I think it could be clearer since Beck missed it, and I did too the first time I looked at it. Would this be clearer?

>A temporary ban from the community, including any sort of interaction or public communication, for a specified period of time.

**Comment by tpendragon** [comment]
If there's a difference, can we just call it good? It makes it easier to maintain if it's mostly copy/paste from https://www.contributor-covenant.org/version/2/1/code_of_conduct/

**Comment by escowles** [comment]
Yes, I am fine with the text as-is.

**Review by escowles — APPROVED** [review]

**Comment by tpendragon** [comment]
@Beck-Davis I'm interested in your feedback here. From my point of view using the text as given from the CoC site makes our lives a little easier, but you're right that the sentence in phrase 3 is easier to read. Is the sentence in phrase 2 confusing enough to you that in your opinion we should take on that diff?

If so, maybe we should have two commits on this PR - one with the copy/pasted text and one with the changes, so we can keep track of what we've changed?

Edit: Oh, it looks like that's what's happened. Thanks Max!

Edit: Yeah we've already taken on diffs, I retract my concern. :+1: to @Beck-Davis's proposal.

**Comment by Beck-Davis** [comment]
@tpendragon I was looking back over the commits carefully before I commented again. But yeah if your concern is addressed, I think we should take the opportunity to clear up wording. And it looks like you did it just as I was writing this comment. 🎉

**Comment by tpendragon** [comment]
I took a shot at the change: https://github.com/pulibrary/.github/pull/6/commits/6d8b0448fe54b7e366ad3c3492d428bddf345976.

I had to re-add "for a specified amount of time" since there's no implied timeframe in 2, and kept the bit about avoiding interactions in community spaces since as a document I think it's useful to have that somewhere, even if it's not in 3.

**Review by hackartisan — APPROVED** [review]
Thanks for everyone's work on this

**Pull request merged**